internal/backend/auth/authsessions/config.go

```diff
@@ -0,0 +1,30 @@
+package authsessions
+
+import (
+	"github.com/dghubble/sessions"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+)
+
+type Config struct {
+	UnsafeCookie *sessions.CookieConfig `koanf:"cookie"`
```

Comment:
why UnsafeCookie ?

Reply:
i think i meant here that if you specify something here, it doesn't use `sessions` default safe cookie. this is not necessary. renamed to `Cookie`.